docker/ubuntuDockerfile

```diff
@@ -20,16 +22,37 @@ COPY thirdparty thirdparty
 COPY scripts scripts
 COPY requirements requirements
 
-ENV MR_STATE=DOCKER_BUILD
-
 # build and install thirdparty
 RUN sudo ./scripts/build_thirdparty.sh
-RUN sudo ./scripts/install_thirdparty.sh
+
+
+FROM ubuntu:20.04
+
+ENV MR_STATE=DOCKER_BUILD
```

Comment:
do we need to set it twice (line 14)

Reply:
yes, these are two different builds: first on lines 1 - 26, and second starts on the line 29